samples/javascript_nodejs/10.prompt-validations/prompts/dobPrompt/index.js

```diff
@@ -7,7 +7,7 @@ const DATE_LOW_BOUNDS = new Date('1918-08-24');
 const DATE_HIGH_BOUNDS = new Date('2018-08-24');
 
 // This is a custom DateTimePrompt that requires the date to be between DATE_LOW_BOUNDS and DATE_HIGH_BOUNDS.
-module.exports = class DOBPrompt extends DateTimePrompt {
+module.exports.DobPrompt = class DobPrompt extends DateTimePrompt {
     constructor(dialogId) {
         super(dialogId, async (prompt) => {
             try {
```

Comment:
We should add a comment here to point readers to our text-recognizers suite .. 
https://github.com/Microsoft/Recognizers-Text/tree/master/JavaScript/packages/recognizers-text-suite

may be post ignite we can update this to show using the Timex and resolvers with constraints to have more impactful validation rules.. 

See [here](https://github.com/Microsoft/BotBuilder-Samples/blob/vishwac/cafebotv2/javascript_nodejs/50.contoso-cafe-bot/dialogs/shared/stateProperties/reservationProperty.js) for an usecase in cafebot.

Reply:
good idea, it is an impressive library